prov/usnic/src/usdf_cm.c

```diff
@@ -688,6 +755,11 @@ bool usdf_cm_addr_is_valid_sin(void *addr, size_t addrlen, uint32_t addr_format)
 			return false;
 		}
 		return true;
+	case FI_ADDR_STR:
```

Comment:
I don't think this should be here, this path is only taken if we're checking the address for compatibility with `FI_SOCKADDR[_IN]`, right? 

In any case, I don't see it being used on `FI_ADDR_STR` anywhere.